src/main/java/use_cases/display_curriculums/DisplayCurriculumsPresenter.java

```diff
@@ -0,0 +1,23 @@
+package use_cases.display_curriculums;
+
+import java.util.List;
+
+public class DisplayCurriculumsPresenter implements DisplayCurriculumsOutputBoundary {
+
+    private final List<DisplayCurriculumsInterface> viewObservers;
+
+    public DisplayCurriculumsPresenter(List<DisplayCurriculumsInterface> viewObservers) {
```

Comment:
Good use of dependency inversion here! This should definitely help us adhere to clean architecture. I remember that circular issue we had, and this was such a good way to solve it while making sure our Controllers adhere to SRP. Good job!